src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/licence/Licence.kt

```diff
@@ -44,28 +43,28 @@ data class Licence(
   @Schema(
     description = "The date and time that the prison approved the licence, where licences approved before 01/04/2022 " +
       "will not have an approved time",
-    example = "24/08/2022 11:30:33",
+    example = "2022-08-24T10:30:33",
   )
-  @JsonFormat(pattern = "dd/MM/yyyy HH:mm:ss")
+  @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss")
   val approvedDateTime: LocalDateTime,
 
   @Schema(description = "The username of the person who created the licence", example = "X12333")
   val createdByUsername: String,
 
-  @Schema(description = "The date and time that the licence was first created", example = "24/08/2022 09:30:33")
-  @JsonFormat(pattern = "dd/MM/yyyy HH:mm:ss")
-  val createdDate: LocalDateTime,
+  @Schema(description = "The date and time that the licence was first created", example = "2022-08-24T11:30:33")
+  @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss")
+  val createdDateTime: LocalDateTime,
 
   @Schema(description = "The username of the person who last updated the licence", example = "X34433")
   val updatedByUsername: String,
 
-  @Schema(description = "The date and time that the licence was last updated", example = "24/08/2022 09:30:33")
-  @JsonFormat(pattern = "dd/MM/yyyy HH:mm:ss")
-  val updatedDate: LocalDateTime,
+  @Schema(description = "The date and time that the licence was last updated", example = "2022-08-24T12:30:33")
+  @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss")
+  val updatedDateTime: LocalDateTime,
```

Comment:
this is nullable in the database and seems be null in prod for quite a few licences. I'm not sure how as it is set when you create the licence and I can't see where it gets nulled.

Reply:
it makes sense conceptually for it to be null. I'll update this